docs/features/spellcheck/README.md

```diff
@@ -92,9 +93,29 @@ For more detailed documentation, refer to the [official WProofreader "Getting St
 
 ## Supported Languages
 
-By default the WebSpellChecker solutions support 17 languages for spell checking: American English, British English, Brazilian Portuguese, Canadian English, Canadian French, Danish, Dutch, Finnish, French, German, Greek, Italian, Norwegian Bokmal, Portuguese, Spanish, Swedish and Ukrainian. Grammar checking is available for 15 of them &mdash; there is no grammar checking for Finnish and Norwegian.
+### Language support
+
+The most popular languages used with WProofreader include: American English, British English, Canadian English, Canadian French, French, German, Italian, Greek, Spanish, Finnish, Danish, Dutch, Portuguese, Swedish, Ukrainian, Norwegian Bokmål, Brazilian Portuguese. There are, however, over 160 languages altogether, available for download from the WebSpellChecker site. Grammar checking is available for over 20 languages.
+
+A recent addition to the software are AI-driven tools. Smart algorithms employed in the AI-based language dictionaries offer a far better checking quality, generating proofreading suggestions based on the context of the sentence. They provide more suitable suggestions that address mistakes with thrice the accuracy of traditional dictionaries. The AI-based support is currently available for English and German.
+
+Here you can check the [full list of available languages](https://webspellchecker.com/additional-dictionaries/).
+
+### Custom dictionaries
+
+Apart from the language dictionaries, WebSpellChecker offers two specialized dictionaries: a **medical** and a **legal** one, available for an additional fee.
+
+There are also two types of custom dictionaries available.
+
+One is the **user dictionary** that can be expanded and will grow during the regular use of the feature simply by adding new words to the dictionary. This is a perfect solution for editors and writers working on specific content that may contain slang or professional jargon.
+
+The other is the so-called **company-wide dictionary**. These premade dictionaries can be uploaded by the system administrators or CKEditor 5 integrators and can be made available company-wide, accessible for all users. This way all benefits of a user-generated dictionary can be shared among the team, making the proofreading process more structured and controlled.
+
+### Multi-language Support
+
+For content in multiple languages, it is also possible to use WProofreader. The spell check feature can automatically detect the text language properly. It is enough to choose “Auto Detect” in the Language dropdown in the WProofreader settings. Suggestions for spelling and grammar (where available) will be displayed properly for each text fragment.
```

Comment:
As a matter of fact, this is supported in both WProofreader and SCAYT.

Reply:
Let's remove the first sentence, then.